docs/data_management/lts/top_level_lts_permission.md

```diff
@@ -0,0 +1,63 @@
+
+- **Stewards**: Stewards are individuals responsible for managing an allocation. Stewards need a full access key pair to perform tasks like creating, deleting, and maintaining buckets. Each steward must maintain separate key pairs for their allocations and any lab/Core allocations they manage.
+- **Lab PIs**: Lab PIs manage the storage allocations for their Labs. They are responsible for ensuring compliance with data management policies and may assign stewards to assist with managing the allocations.
+- **Core Director**: Core Directors manage the storage allocations for Core facilities. They also ensure compliance with data management policies, and designate stewards, if necessary, to assist with the management of the allocation.
+- **Bucket Policy**: A bucket policy is a [JSON formatted file](https://docs.fileformat.com/web/json/) that you can use to grant access permissions to your LTS bucket and the objects in it. Please refer to [sharing buckets and bucket policy structure](../lts/policies.md#sharing-buckets) for more details.
+
+## Shared LTS Allocations
+
+Shared LTS allocations are storage spaces designed for collaborative use by groups and are available to Lab PIs and Core Directors. If a PI is also a Core Director, they are eligible for independent storage allocations for each organization: one for the Lab and one for the Core. The shared LTS allocation provides 75 TB of storage. These allocations can be named according to the preference of the PI or Core Director. For recommended naming guidelines, refer to our [Naming Shared Storage](../../data_management/storage.md#how-do-i-request-shared-storage) documentation.
+
+{{ read_csv('data_management/res/shared_allocation_functional_roles.csv', keep_default_na=False) }}
+
+### Who can have what role?
+
+- Owners must be Core Directors or Lab PIs.
+- Stewards may be any UAB affiliated person designated by an owner.
+- Members may be anyone with access to the system, i.e., someone who has created an individual LTS allocation and has a Cheaha account.
+
+### How do I gain access to my role?
+
+- Owners and stewards have key sets for the allocation(s) they manage. These key sets are distinct, one per person per allocation, and separate from the key set they use for their individual allocations.
+- Members are granted access by [bucket policy](https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucket-policies.html) for each bucket they need access to. For more details, please refer to our documentation on [sharing buckets and bucket policy structure](../lts/policies.md#sharing-buckets).
+
+## Individual LTS Allocations
+
+Individual LTS allocations are intended for personal or individual use and are available to all UAB affiliated individuals. These allocations are tied to the individual’s email and provide 5 TB of storage.
+
+{{ read_csv('data_management/res/individual_allocation_functional_roles.csv', keep_default_na=False) }}
+
+### How do I grant other users access to my Allocation?
```

Comment:
Address this question as a subsection in the "Shared LTS Allocations" section. We need to add clarity to the possibility of steward roles and what they are used for.

Add this subsection before the other questions. Then use it to segue into the idea of roles, which will then be addressed naturally as the reader progresses.